doc/source/api/quickstart.rst

```diff
@@ -0,0 +1,185 @@
+Quickstart
+==========
+
+Log in as your user, navigate to the API Keys page in the User menu, and
+generate a new API key.  Make a note of the API key, and then pull up a
+terminal.  Now we'll use the display.py script in your galaxy/scripts/api
+directory for a short example::
+
+        % ./display.py my_key http://localhost:4096/api/histories
+        Collection Members
+        ------------------
+        #1: /api/histories/8c49be448cfe29bc
+          name: Unnamed history
+          id: 8c49be448cfe29bc
+        #2: /api/histories/33b43b4e7093c91f
+          name: output test
+          id: 33b43b4e7093c91f
+
+The result is a Collection of the histories of the user specified by the API
+key (you).  To look at the details of a particular history, say #1 above, do
+the following::
+
+        % ./display.py my_key http://localhost:4096/api/histories/8c49be448cfe29bc
+        Member Information
+        ------------------
+        state_details: {'ok': 1, 'failed_metadata': 0, 'upload': 0, 'discarded': 0, 'running': 0, 'setting_metadata': 0, 'error': 0, 'new': 0, 'queued': 0, 'empty': 0}
+        state: ok
+        contents_url: /api/histories/8c49be448cfe29bc/contents
+        id: 8c49be448cfe29bc
+        name: Unnamed history
+
+This gives detailed information about the specific member in question, in this
```

Comment:
Because you are very used to the REST terminologies. In documentation I'd rather spend one extra word and be clear, than make the reader wonder: "member of what?"